src/muse/errors.py

```diff
@@ -0,0 +1,31 @@
+class RetrofitAgentNotDefined(Exception):
+    """Indicates that the retrofit agent has not been defined."""
+
+    msg = """The retrofit agent has not been defined. This might be because it actually
+has not been defined in the agents file or because it cannot be found because there is a
+typo in its name in either the agents and or technodata files."""
+
```

Comment:
Update msg
msg = = """The retrofit agent has not been defined. This might be because it actually
has not been defined in the agents file or specified in the technodata file or it cannot be found because there is a
typo in its name in either the agents and or technodata files. When a retrofit agent is defined in the agent file, the same name should be reported as an additional column in the technodata, assigning the fraction of the initial stock assigned to each retrofit agent. In presence of multiple retrofit agents, a new column needs to be added per agent."""